src/modules/user/routes.js

```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+import CreateUserValidation from '../../middlewares/CreateUserValidationSchema';
+import IsAuthenticatedMiddleware from '../../middlewares/IsAuthenticatedMiddleware';
+import create from './user.controller';
+import checkPermissions from '../../middlewares/checkPermissionsMiddleware';
+
+const router = express.Router();
+
+router.post('/create', IsAuthenticatedMiddleware, checkPermissions('REGISTER_USERS'), CreateUserValidation, create);
```

Comment:
I believe we are following the `REST API` convention.
For creating records, we would do a POST to `api/users/` and not `api/users/create`